samples/nrf9160/lwm2m_client/scripts/coiote.py

```diff
@@ -103,11 +163,86 @@ def delete_device(self, dev_id):
         else:
             logging.info('Coiote: Deleted device %s', dev_id)
 
-    def create_device(self, dev_id, psk):
+    def batch_create_device(self, dev_file, model, lines):
+        # dev_list is in form:  \"352656100030868\",\"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\"
+
+        logging.info(f"URL = {self.api_url}")
+
+        line_cnt = 0
+        line_tot = 0
+        succeeded = 0
+        failed = 0
+
+        with open(dev_file,"r") as fh:
+            objs = []
+            domain = self.get_domain()
+            
+            for line in fh:
+                dev_id = f"urn:imei:{line[1:16]}"
```

Comment:
The prefix could be configurable

Reply:
And maybe split on comma instead of counting characters.

Reply:
Will do both proposals.